tools/format.py

```diff
@@ -68,13 +69,14 @@ def rustfmt():
         rustfmt_config,
     ] + find_exts(["cli", "core", "tools", "deno_typescript", "cli_snapshots"],
                   [".rs"],
-                  skip=["core/libdeno/build"]))
+                  skip=["core/libdeno/build", "core/libdeno/third_party"]))
 
 
 def gn_format():
     print "gn format"
-    for fn in find_exts(["core/libdeno"], [".gn", ".gni"],
-                        skip=["core/libdeno/build"]):
+    for fn in find_exts(
+        ["core/libdeno"], [".gn", ".gni"],
+            skip=["core/libdeno/build", "core/libdeno/third_party"]):
```

Comment:
This still touches files in the build and third_party submodule and idk why. Please help.